api/maintainer/owned-packages.js

```diff
@@ -15,7 +15,12 @@ module.exports = async (req, res, ctx) => {
     const cleanPackages = packages.map(({
       id, name, language, registry, donationRevenue, adRevenue
     }) => ({
-      id, name, language, registry, donationRevenue, adRevenue
+      id,
+      name,
+      language,
+      registry,
+      donationRevenue: donationRevenue ? donationRevenue.reduce((acc, r) => (acc += r.amount), 0) : 0,
```

Comment:
although `+=` does return the mutated var, I think it's a little confusing and pretty close to non functional code (returning an assignment). can we change these reduces to `(acc, r) => acc + r.amount` ?

Reply:
rogggger